### ISSUE TITLE
Telegram Messenger

### ISSUE THREAD

**Issue by HeimLabor** [vitalidze/traccar-web#744]
Hello Vitaly,

maybe it is a good idea to integrate Telegram Messenger instead or beside of Pushbullet.
Telegram is more flexible, you are able to build groups, maybe for the future send messages back to the server. And there is a client for every popular os.

The Messenger has also a simple http API like pushbullet (The User must figure his own bot and chat ID )
`https://api.telegram.org/bot<API-Key-from-Bot>/sendMessage?chat_id=<chatid>text=Text to send here`

Description how bots work -> [https://core.telegram.org/bots](url)

**Comment by vitalidze** [comment]
Well, I think it's a good idea :) I just don't have time to implement it for now. You are welcome to implement it and send a pull request.

**Comment by leontino** [comment]
That's nice! Whatsapp dont have this api?

**Comment by vitalidze** [comment]
Whatsapp does not have official API.

**Comment by 100n0m3** [comment]
you can always use yowsup but they will block your number.

**Comment by vitalidze** [comment]
@100n0m3 yeah, I read about it and there is a PHP library too. First of all, yowsup is written in python, so it will require either third party application somewhere or python interpreter written in java. And though it's not an official way I doubt it is a good idea to include it in the project.

**Comment by 100n0m3** [comment]
here an example of yowsup that can be used

yowsup-cli demos -l number:password -s numebrtosend " example some notification etc.."

number to send could be get of database and the other variables too device etc... just like email but only change the command to shell execute the yowsup.

**Comment by vitalidze** [comment]
This makes things much more complicated. For example on windows OS I doubt the process will be this trivial. 

You are free to implement this method of sending notifications if you want.

**Comment by Nitrosito** [comment]
More info in the  https://github.com/traccar/traccar/issues/4274 and https://github.com/traccar/traccar/commit/22436a2434faa8912c8a352e65306d0dc0669b5c

I vote for implement telegram notification
